chainer/distributions/normal.py

```diff
@@ -32,14 +32,29 @@ class Normal(distribution.Distribution):
         location :math:`\\mu`. This is the mean parameter.
         scale(:class:`~chainer.Variable` or :class:`numpy.ndarray` or \
         :class:`cupy.ndarray`): Parameter of distribution representing the \
-        scale :math:`\\sigma`.
+        scale :math:`\\sigma`. Either `scale` or `log_var` (not both) must \
+        have a value.
+        log_var(:class:`~chainer.Variable` or :class:`numpy.ndarray` or \
+        :class:`cupy.ndarray`): Parameter of distribution representing the \
+        scale :math:`\\log(\\sigma^2)`. Either `scale` or `log_var` (not \
+        both) must have a value.
 
     """
 
-    def __init__(self, loc, scale):
+    def __init__(self, loc, scale=None, log_var=None):
```

Comment:
Would you like to make `log_var` keyword-only?  (If so, use `chainer.utils.argument.parse_kwargs`).